examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>typo: whererever should be wherever</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>